assets/components/Network.vue

```diff
@@ -266,6 +266,14 @@ export default {
     this.visData.edges = mountVisData(this, 'edges');
     this.network = new Network(container, this.visData, this.options);
 
+    this.network.on( 'doubleClick', function(properties) {
+      var adm_slug = properties.nodes.toString();
+      if (adm_slug.indexOf('Abstract') !== -1) {return}  //the abstract property should be inside properties
+      var n = adm_slug.lastIndexOf(".");
+      adm_slug = adm_slug.substring(n+1).replace("/", ".");
+      window.open("/admin/doc/models/" + adm_slug, '_blank');
```

Comment:
This being hard-coded means it should at least be documented in the README that it must be that path.

Ideally some use of `reverse(..)` would occur so that it isnt hard-coded.

fwiw, the double click isnt working for me.  I havent worked out why yet.